RadixWallet/Core/FeaturePrelude/AddressView/AddressDetails.swift

```diff
@@ -1,18 +1,21 @@
 // MARK: - AddressDetails
 public struct AddressDetails: Sendable, FeatureReducer {
```

Comment:
You can add `@Reducer` to this. Then you need to also put `public typealias Action = FeatureAction<Self>` explicitly, otherwise the compiler gets confused.

Reply:
and what's the benefit of doing so if we still need to implement every `func reduce` we currently have?

Reply:
It's usually good to stay up to date with the latest generation of TCA, so we can enjoy any future improvements coming to `@Reducer` they make.